govcd/api_test.go

```diff
@@ -22,7 +22,7 @@ var INVALID_NAME = `*******************************************INVALID
 					************************`
 
 // This ID won't be found by lookup in any entity
-var invalidEntityId = "one:two:three:aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+var invalidEntityId = "urn:vcloud:three:aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
```

Comment:
In some cases (Query filtering) filters by ID return HTTP 500 without valid URN beginning